src/psyclone/generator.py

```diff
@@ -729,7 +729,7 @@ def code_transformation_mode(input_file, recipe_file, output_file,
             Profiler.add_profile_nodes(routine, Loop)
 
         # Generate Fortran
-        output = FortranWriter()(psyir)
+        output = FortranWriter(disable_copy=True)(psyir)
```

Comment:
Please extend the comment to say why we 'disable_copy' i.e. this is the final step and so the tree cannot subsequently be modified.

Reply:
Done